common/configs/src/eth.rs

```diff
@@ -27,13 +27,14 @@ use serfig::parsers::Toml;
 use crate::LogConfig;
 use crate::StorageConfig;
 
-#[derive(Parser, Debug, Default, Clone, Serialize, Deserialize)]
+#[derive(Parser, Debug, Clone, Serialize, Deserialize)]
```

Comment:
```suggestion
#[derive(Parser, Debug, Clone, Serialize, Deserialize)]
#[serde(default)]
```

All config should have `#[serde(default)]` to make loading from files work.